health/healthService_test.go

```diff
@@ -0,0 +1,27 @@
+package health
+
+import (
+	"testing"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	_ = logger.SetLogLevel("health:TRACE")
+}
+
+func TestNewHealthService(t *testing.T) {
+	h := NewHealthService(config.HealthServiceConfig{}, "")
+	require.NotNil(t, h)
+}
+
+func TestHealthService_StartThenClose(t *testing.T) {
+	h := NewHealthService(config.HealthServiceConfig{}, "")
+	require.NotNil(t, h)
+
+	h.Start()
+	err := h.Close()
+	require.Nil(t, err)
+}
```

Comment:
More unit tests?

Reply:
Added. I've sketched one way to mock the time, if it looks good perhaps we can use it in some of our long-running integration tests.

Reply:
This is the way it can be tested: mock everything that involves timing and, in more general, everything is I/O :+1: